src/Logary/Targets_Graphite.fs

```diff
@@ -71,7 +71,7 @@ module Impl =
   /// All graphite messages are of the following form.
   /// metric_path value timestamp\n
   let createMsg (path : String) value (timestamp : Instant) =
-    let line = String.Format("{0} {1} {2}\n", path, value, timestamp.Ticks / NodaConstants.TicksPerSecond)
+    let line = String.Format("{0} {1} {2}\n", path, value, (timestamp |> Internals.Date.getTicks ) / NodaConstants.TicksPerSecond)
```

Comment:
It's alright since v4 is still in beta. I haven't frozen the API yet, but we're really getting there. The last couple of weeks have brought tremendous improvements (IMO).